ntc_templates/templates/extreme_slxos_show_ip_interface_brief.textfsm

```diff
@@ -0,0 +1,13 @@
+Value INTERFACE ((Port-channel|Loopback|Ethernet|Ve) \d+(?:/\d+)?(?:\s*\(I\))?)
+Value IP_ADDRESS (\S+)
+Value VRF (\S+)
+Value STATUS (\S+\s?\S+)
+Value PROTOCOL (\S+)
+
+Start
+  ^\s*Flags
+  ^\s*Interface
```

Comment:
```suggestion
  ^\s*Interface\s+IP-Address\s+Vrf\s+Status\s+Protocol\s*$$
```
We want to capture all of the headers to make sure the data is what fields we are expecting